versioned_docs/version-3.0.0-beta/framework-guide/deploy-nuxt.md

```diff
@@ -0,0 +1,65 @@
+---
+id: deploy-nuxt
+title: How to Deploy a Nuxt.js Site with Spheron
+sidebar_label: Deploy Nuxt.js App
+slug: /framework-guide/deploy-nuxt
+---
+
+This guide will show you how to deploy a Nuxt.js project and get your domain up.
+
+[Nuxt.js](https://nuxtjs.org/) is a free and open source JavaScript library based on Vue.js, Node.js, Webpack and Babel.js. Nuxt is inspired by Next.js, which is a framework of similar purpose, based on React.js. The framework is advertised as a "Meta-framework for universal [applications"](https://en.wikipedia.org/wiki/Nuxt.js).
+
+
+## Follow these steps to deploy your Nuxt.js app on Spheron.
+
+### Step 1: Put Your Nuxt.js Project Code in a Git Repo
+
+While this feature is not new to developers, the magic of Spheron is that every push to any branch in your GitHub, GitLab, or Bitbucket repo triggers a new build and deployment of your project.
+
+### Step 2: Update Your Nuxt.js Project for Deploying on Decentralised Storage
```

Comment:
This is completely wrong, use the next js part for nuxt app. Please do some research before writing. This can cause a lot of issue for users who are trying to deploy their app and follow something completely wrong.

Reply:
line 17 to 18 is correct. are you saying a nuxt.js project don't need the hash routing?

Reply:
First fix for path can be done using this -https://nuxtjs.org/docs/configuration-glossary/configuration-router/#trailingslash

If it still doesn't work then use hash routing but the way to enable is given in the doc is completely wrong, please check this - https://nuxtjs.org/docs/configuration-glossary/configuration-router/#mode